pkgs/by-name/ni/nixos-rebuild-ng/README.md

```diff
@@ -101,6 +101,44 @@ ruff check --fix .
 ruff format .
 ```
 
+## Breaking changes
+
+While `nixos-rebuild-ng` tries to be as much of a clone of the original as
+possible, there are still some breaking changes that were done in other to
+improve the user experience. If they break your workflow in some way that is
+not possible to fix, please open an issue and we can discuss a solution.
+
+- For `--build-host` and `--target-host`, `nixos-rebuild-ng` does not allocate
+  a pseudo-TTY via SSH (e.g.: `ssh -t`) anymore. The reason for this is because
+  pseudo-TTY breaks some expectations from SSH, like it mangles stdout and
+  stderr, and can [break terminal
+  output](https://github.com/NixOS/nixpkgs/issues/336967) in some situations.
```

Comment:
```suggestion
  stderr, and can [break terminal output](https://github.com/NixOS/nixpkgs/issues/336967) in some situations.
```
please keep links searchable otherwise you'll never find it from a rendered doc :)